### PR TITLE
Update workflows to use octorelease

### PR DESCRIPTION
**Pull request by awharn**
Signed-off-by: Andrew W. Harn <andrew.harn@broadcom.com>

### CODE REVIEW

File: .github/workflows/zowe-cli-plugin.yml [hunk 1]
```diff
@@ -40,11 +34,9 @@ jobs:
       run: npm install -g npm@7
 
     - name: Install Node Package Dependencies
-      id: install
       run: npm ci
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:37]: should we call the `zowe-actions/octorelease-script` `npmUpdate` script right after `npm ci` ?

**Review by zFernand0 — COMMENTED** [review]
Looking good so far.
curious if you'd like to fork the repo and test it out with GH Packages ?
Or maybe publish a `update-pipeines` prerelease tag to zowe jfrog ?

File: .github/workflows/zowe-cli-plugin.yml (hunk 1 above)
Comment (awharn) [reply .github/workflows/zowe-cli-plugin.yml:37]: We probably should, and I am going to update it.

**Review by zFernand0 — APPROVED** [review]
Thanks for testing! 😋

**Review by gejohnston — APPROVED** [review]

**Pull request merged**